xml_converter/generators/code_generator.py

```diff
@@ -359,13 +388,55 @@ def build_example(self, type: str, applies_to: List[str], xml_field: str, exampl
 
         return example
 
+    ############################################################################
+    # Generate Node Types
+    #
+    # This will output code for a single category of nodes.
+    ############################################################################
+    def generate_node_types(self, metadata: Any, attributenames: Dict[str,str],page: str) -> Tuple[List[str],List[str]]:
+        
+        field_rows: List[str] = []
+        includelist: Set[str] = set()
```

Comment:
Lets be slightly more descriptive with this name and call it something like `cpp_include_files`, `cpp_include_paths`, `cpp_include_libraries`, or `cpp_attribute_libraries`.

Reply:
changed to 'cpp_include_paths'